swagger.yaml

```diff
@@ -69,17 +68,203 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Tasks"
-  
-   
+
+  "/salesLeads":
+    get:
+      operationId: listSalesLeads
+      summary: Lists all Leads from Pipedrive
+      description: |
+        Lists Leads from Pipedrive
+      tags:
+        - Leads
+      responses:
+        "200":
+          description: Lists Leads from Pipedrive
+          content:
+            application/json;charset=utf-8:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Lead"
+
+  "/salesDeals/{status}":
```

Comment:
Is it necessary for status to be required here?

Reply:
Yes for now